frameworks/Haskell/yesod-postgres/bench/.gitignore

```diff
@@ -0,0 +1,8 @@
+dist*
+static/tmp/
+config/client_session_key.aes
+*.hi
+*.o
+*.sqlite3
+.hsenv*
+yesod-devel/
```

Comment:
Made the change. Can't add `bench` to `.gitignore` because `bench/src` contains the source code.